lighthouse-core/test/audits/load-fast-enough-for-pwa-test.js

```diff
@@ -78,4 +78,37 @@ describe('PWA: load-fast-enough-for-pwa audit', () => {
     expect(result.rawValue).toBeGreaterThan(2000);
     expect(Math.round(result.rawValue)).toMatchSnapshot();
   });
+
+  it('overrides when throttling is modified but method is not provided', async () => {
```

Comment:
this test name doesn't seem to line up with what's happening isn't the method provided as devtools here?